app/models/manageiq/providers/autosde/storage_manager/physical_storage.rb

```diff
@@ -1,5 +1,12 @@
 class ManageIQ::Providers::Autosde::StorageManager::PhysicalStorage < ::PhysicalStorage
   supports :create
+  supports :delete
```

Comment:
@llivne and here you can add your check that it is connected to an EMS:
```
supports :delete do
  unsupported_reason_add(:delete, _("The Physical Storage is not connected to an active Manager"))
end
```

Reply:
We don't disconnect physical storages when an EMS is deleted though so unless someone manually created the record or deleted the EMS without calling destroy this can't happen (a physical storage without an EMS).